src/components/Tooltip/index.native.js

```diff
@@ -0,0 +1,8 @@
+import {propTypes, defaultProps} from './TooltipPropTypes';
+
```

Comment:
Please add some comments to this file to explain the differences between this and the web version (and more importantly _WHY_ it should be different).